website/content/commands/debug.mdx

```diff
@@ -71,14 +71,14 @@ all targets for 2 minutes.
-| `metrics` | Metrics from the in-memory metrics endpoint in the target, captured at the interval.                                                                                                |
-| `logs`    | `DEBUG` level logs for the target agent, captured for the interval.                                                                                                                 |
-| `pprof`   | Golang heap, CPU, goroutine, and trace profiling. This information is not retrieved unless [`enable_debug`](/docs/agent/options#enable_debug) is set to `true` on the target agent. |
+| Target    | Description                                                                                                                                                                                                                                                                           |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `agent`   | Version and configuration information about the agent.                                                                                                                                                                                                                                |
+| `host`    | Information about resources on the host running the target agent such as CPU, memory, and disk.                                                                                                                                                                                       |
+| `cluster` | A list of all the WAN and LAN members in the cluster.                                                                                                                                                                                                                                 |
+| `metrics` | Metrics from the in-memory metrics endpoint in the target, captured at the interval.                                                                                                                                                                                                  |
+| `logs`    | `DEBUG` level logs for the target agent, captured for the interval.                                                                                                                                                                                                                   |
+| `pprof`   | Golang heap, CPU, goroutine, and trace profiling captured for `-duration` time. This information is not retrieved unless [`enable_debug`](/docs/agent/options#enable_debug) is set to `true` on the target agent or ACL are enable and an ACL token with `operator:read` is provided. |
```

Comment:
I'm not sure if this is still accrurate

Reply:
@dnephin this is still true but not fully accurate, do you think we should say more in the doc or this is the right level of detail?

Reply:
This is really the most detailed reference documentation we have for these features so I think this would be a good place to be precise about what users should expect. I don't think we need to go into implementation specifics but I think this would be a good place to describe that `profile` and `trace` will now be single captures for the full duration while `heap` and `goroutines` will be captured per-interval.

Reply:
I updated it with more details, let me know if this is ok